docs/CDCBigTable-sparksink.md

```diff
@@ -1,16 +1,45 @@
-CDC Google Bigtable Sink
-==========
+# CDC Google Bigtable Sink
```

Comment:
Looks like the official name is "Cloud Bigtable", so let's change "Google Bigtable" to "Google Cloud Bigtable" here, and call it "Cloud Bigtable" everywhere after this

Reply:
fixed